### PR TITLE
SOLAR HOT WATER SYSTEM ADDITION

### PR DESCRIPTION
**Pull request by darshanjain12**

### CODE REVIEW

File: ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -34,4 +34,14 @@ LVLM:
   - level_status
   opt_uses:
   - percentage_sensor
+
+WTS:
```
Comment (tasodorff) [inline ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml:38]: this is called `WDT` in the HVAC namespace, so i would suggest using that name.  we can eventually elevate this concept to the global namespace and reference it in local namespaces, all under the same name.

**Review by tasodorff — CHANGES_REQUESTED** [review]
Question on namespace appropriateness and request to adjust name of abstract type.

File: ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (darshanjain12) [reply ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml]: Yes. Committed the changes to add in the global name space

Comment (darshanjain12) [reply ontology/yaml/resources/PLUMBING/entity_types/ABSTRACT.yaml]: Closing the pull request as we will be using HWS_SWTM canonical type.

**Pull request closed without merging**